lib/JsonpMainTemplate.runtime.js

```diff
@@ -3,49 +3,52 @@
 	Author Tobias Koppers @sokra
 */
 /*globals hotAddUpdateChunk parentHotUpdateCallback document XMLHttpRequest $require$ $hotChunkFilename$ $hotMainFilename$ */
+"use strict";
```

Comment:
Thanks for the feedback. I'm very new to the project and it helps me a lot! I will revert the commit and there's also another thing in the `ContextModuleFactory.js` I have to fix.